### PR TITLE
HTML: Add a historical test for the search event and incremental=""

### PR DESCRIPTION
**Pull request by zcorpan**
See https://github.com/whatwg/html/issues/667

### CODE REVIEW

File: html/semantics/forms/historical-search-event.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<input id=input type=search incremental>
+<script>
+promise_test(async t => {
+  const input = document.getElementById('input');
+  const eventWatcher = new EventWatcher(t, input, ['search', 'keypress']);
+  await Promise.all(
```
Comment (domenic) [inline html/semantics/forms/historical-search-event.html]: Promise.all takes an array, here and below.

File: html/semantics/forms/historical-search-event.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    test_driver.send_keys(input, 'x'),
+    eventWatcher.wait_for(['keypress'])
+  );
+  // During this timeout, the search event will fire, if it's supported,
+  // which fails the test since the event watcher isn't expecting it.
+  await new Promise(resolve => t.step_timeout(resolve, 1000));
+  await Promise.all(
```
Comment (domenic) [inline html/semantics/forms/historical-search-event.html]: What is the purpose of this step?

File: html/semantics/forms/historical-search-event.html (hunk 1 above)
Comment (zcorpan) [reply html/semantics/forms/historical-search-event.html]: Oooops, fixed.

File: html/semantics/forms/historical-search-event.html (hunk 2 above)
Comment (zcorpan) [reply html/semantics/forms/historical-search-event.html]: Only to increase my confidence that the test works, but it's not necessary.

**Comment by zcorpan** [comment]
Test results:
https://wpt.fyi/results/html/semantics/forms?label=pr_head&max-count=1&pr=19889

**Review by domenic — APPROVED** [review]
LGTM as tests, and I guess we should feel free to merge tests that reflect the current spec. But it's not clear to me what the future is for these; in particular the search event seems like it might need to get specced based on the use counter discussions.

**Comment by zcorpan** [comment]
@tkent-google do you think this should be merged now or wait until we know if removal is successful?

**Review by tkent-google — APPROVED** [review]
> do you think this should be merged now or wait until we know if removal is successful?

I think Chrome can remove them, and we can merge this PR now.

**Comment by zcorpan** [comment]
Thanks @tkent-google

**Pull request merged**